app/views/user_course_subjects/show.html.erb

```diff
@@ -0,0 +1,21 @@
+<h1><%= t :h1_show_user_course_subjects %></h1>
+<table class="table">
+  <thead>
+  <tr>
+    <th scope="col"><%= t :name_show_user_course_subjects %></th>
+    <th scope="col"><%= t :status_show_user_course_subjects %></th>
+    <th scope="col"><%= t :created_at_show_user_course_subjects %></th>
+  </tr>
+  </thead>
+  <tbody>
+    <% load_subject(params[:id]) %>
+    <% @subjects_all.each do |n| %>
+      <tr>
+        <%load_name_subject n.course_subject_id%>
```

Comment:
line này để làm j em, chị ko thấy em hiển thị

ngoài ra, tìm hiểu về `includes` đi em ko cần phải load thủ công từng cái thế này

Reply:
ở đây chủ yếu em muốn load ra name của subject  để in ra màn hình ạ

includes: dùng để load ra các giá trị một cách nhanh hơn